doc_conf/index.rst

```diff
@@ -5,7 +5,7 @@
 
 HiDimStat: High-dimensional statistical inference tool for Python
 =================================================================
-|Build| |codecov| |codestyle|
+|Build| |CircleCI/Documentation| |codecov| |codestyle|
```

Comment:
I think you also need to add ..|CircleCI/Documentation| + URL at the end of the file

Reply:
thank you for the hint.